### PR TITLE
lmp/bb-config: use EXTRA_IMAGE_INSTALL when they have content

### PR DESCRIPTION
**Pull request by quaresmajose**
Signed-off-by: Jose Quaresma <jose.quaresma@foundries.io>

### CODE REVIEW

File: lmp/bb-config.sh [hunk 1]
```diff
@@ -135,14 +135,16 @@ EOFEOF
 sed -e "s|@@ARCHIVE@@|${archive}|" ${HERE}/bb_logconfig.json > bb_logconfig.json
 
 # Additional packages based on the CI job used
-if [ "$CONF_VERSION" == "1" ]; then
-	cat << EOFEOF >> conf/local.conf
+if [ -n ${EXTRA_IMAGE_INSTALL} ];then
```
Comment (ricardosalveti) [inline lmp/bb-config.sh:138]: Add "" before/after ${EXTRA_IMAGE_INSTALL} to make sure it is always valid. Please also add space after ; and before then.

Comment (quaresmajose) [reply lmp/bb-config.sh]: Thanks but don't get the part to add the space before
```
if [ -n "${EXTRA_IMAGE_INSTALL}" ] ; then
```
is this what you mean?

Comment (ricardosalveti) [reply lmp/bb-config.sh]: Sorry, just before then is enough :-)

`if [ -n "${EXTRA_IMAGE_INSTALL}" ]; then
`

Comment (quaresmajose) [reply lmp/bb-config.sh]: fixed

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**